README.md

```diff
@@ -66,6 +66,14 @@ Try out Solidus with one-click on Heroku:
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/solidusio/solidus)
 
+Alternatively, you can use Docker to run a demo in your local machine. Run the following command to download the image and run it at [http://localhost:3000](http://localhost:3000).
+
+```
+docker run --rm -it -p 3000:3000 kinduff/solidus-demo:latest
+```
+
+The admin can be found at [http://localhost:3000/admin/](http://localhost:3000/admin/), credentials are `admin@example.com` and `test123`.
```

Comment:
Perhaps this could say ```The admin interface can be accessed at [http://localhost:3000/admin/](http://localhost:3000/admin/), the default credentials are `admin@example.com` and `test123`.```, what do you think?